### DISCUSSION TITLE
Digital/lab3/tasks/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#13] [Announcements]
# Digital/lab3/tasks/

https://soc.ustc.edu.cn/Digital/lab3/tasks/

**Comment by Peanut-Tang** [comment]
怎么助教这么喜欢青春无限/xk

**Comment by magichear** [comment]
请问选做第三题，按下红色的`button`之后，需要将sw全部复位（置零）吗？（或者能实现吗）
（前面第三题有提到说那个位置控制复位，但是我没发现它有啥明显的效果>_<）

**Reply by wintermelon008** [reply]
不需要的(●'ω'●) 因为 sw 是来自外界的输入信号，所以无法被 `rst` 信号（也就是按钮）复位。一般来说，`rst` 信号用于复位系统内部的寄存器单元，在我们数码管的题目里则对应 `counter`  和 `seg_id` 两个变量。

**Comment by wintermelon008** [comment]
掩码部分需要指出数码管应当熄灭，而不是显示 0
